.gitmodules

```diff
@@ -875,3 +875,6 @@
 [submodule "vendor/grammars/squirrel-language"]
 	path = vendor/grammars/squirrel-language
 	url = https://github.com/mathewmariani/squirrel-language
+[submodule "vendor/grammars/plugin-vscode"]
+	path = vendor/grammars/plugin-vscode
```

Comment:
The `plugin-vscode` folder  is going to look weird in a directory full of more specific filenames: 😢 

~~~diff
+ ./squirrel-language/     # Good
+ ./data-weave-tmLanguage/ # Good
- ./plugin-vscode/         # BAD 
+ ./sublime-fantom/        # Good
+ ./language-reason/       # Good
~~~
This isn't an issue until another user tries using a grammar repo with a siimilar name: One that would
overwrite...

Reply:
What about naming it `ballerina` or `ballerina-grammar`?

Reply:
Go with `ballerina-grammar`. The former name is [already occupied](https://github.com/ballerinalang/ballerina) by something more important.

Reply:
@Alhadis: Thanks for the tip, I will change the path as suggested.

Reply:
@Alhadis / @pchaigno : I have noticed that some of the languages have used `language-xxx` format. If there are no objections, I would like to rename this to `language-ballerina` as well.

Reply:
That format is used almost exclusively by Atom grammar-packages, though I wouldn't be surprised if it's caught on amongst devs working with other editors. We're nitpicking on a very slight detail, though, so I say go for it. :)